### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by JTS-IL** [presscustomizr/customizr#451]
Hi,
A new law about accessibility is valid in my country from 25/10/2015.
According the law, every commercial website must pass accessibility test at least on level AA.
There are some great plugins that assist accomplishing the task, but lately I notices a new theme tag on wordpress.org : `https://wordpress.org/themes/tags/accessibility-ready/`
I think that Customizr should be upgraded with accessibility features and then have this tag too.
I'm not sure I can help coding, but surely I will help testing :-)

**Comment by Nikeo** [comment]
Thanks for this info Yaacov
Do you have a good plugin in mind for that ?

**Comment by JTS-IL** [comment]
There is a great plugin created by Amit Moreno, RTL ready (actually it's LTR ready :-) )
https://wordpress.org/plugins/accessible-poetry

**Comment by ghost** [comment]
For now I'd recommend using accessibility plugin like : https://wordpress.org/plugins/wp-accessibility/
I'll try to address specific acc. issues reported one by one.
